File: samples/src/main/java/tk/zielony/carbonsamples/widget/ExpansionPanelActivity.java
package tk.zielony.carbonsamples.widget;

import android.os.Bundle;
import android.support.annotation.Nullable;

import tk.zielony.carbonsamples.R;
import tk.zielony.carbonsamples.Samples;
import tk.zielony.carbonsamples.SamplesActivity;

public class ExpansionPanelActivity extends SamplesActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_expansionpanel);

        Samples.initToolbar(this, getString(R.string.expansionPanelActivity_title));
    }
}

File: samples/src/main/java/tk/zielony/carbonsamples/widget/ExpansionPanelActivity.kt
package tk.zielony.carbonsamples.widget

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_expansionpanel.*
import tk.zielony.carbonsamples.SampleAnnotation

import tk.zielony.carbonsamples.R
import tk.zielony.carbonsamples.ThemedActivity
import tk.zielony.randomdata.common.TextGenerator

@SampleAnnotation(layoutId = R.layout.activity_expansionpanel, titleId = R.string.expansionPanelActivity_title)
class ExpansionPanelActivity : ThemedActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        initToolbar()

        expansionPanel_text.text = TextGenerator(5, false).next()
    }
}
